dpnp/dpnp_iface_mathematical.py

```diff
@@ -337,22 +337,24 @@ def diff(input, n=1, axis=-1, prepend=None, append=None):
     Limitations
     -----------
     Input array is supported as :obj:`dpnp.ndarray`.
-    Not supported parameters n, axis, prepend, append.
+    Not supported parameters axis, prepend, append.
```

Comment:
I would write:
```
Parameters ``axis``, ``prepend`` and ``append`` are supported only with default values.
Otherwise the function will be executed sequentially on CPU.
```
because actually the parameters are supported, but in some cases we fallback to `numpy`.